pages/docs/user-docs/docs-bootstrap-image.md

```diff
@@ -142,7 +142,25 @@ Version 2.0
 ```
 
 #### %environment
-Akin to labels, you can add pairs of `VARIABLE` and `VALUE` under environment to be sourced when the container is used as variables in the environment. The entire section is written to a file that gets sourced, so you should generally use the same conventions that you might use in a `bashrc` or `profile`. See <a href="/docs-environment-metadata">Environment and Metadata</a> for more information about these two sections.
+You can add environment variables to be sourced when the container is used in the `%environment` section. The entire section is written to a file that gets sourced, so you should generally use the same conventions that you might use in a `bashrc` or `profile`. 
+
+```
+%environment
+    export VADER=badguy
+    export LUKE=goodguy
+    export SOLO=someguy
+```
```

Comment:
@gmkurtzer can you explain to me again why it's bad to export on the same line as definition? I remember you called it a "bashism" and I don't remember why it's bad - I just tried it in a local /bin/sh and it seemed reasonable. But if it could lead to some weird bug I'd rather not tell users to do it, hehe.